docs/1-tendermint-overview.md

```diff
@@ -0,0 +1,93 @@
+# Tendermint protocol overview
+
+We are going to present a very short overview of the [Tendermint](https://github.com/tendermint) consensus algorithm and the main aspects that are necessary in order to understand the fork accountability algorithm. 
+Please note that this is not an exhaustive and formal description, we invite the reader to check out the [Tendermint documentation](https://docs.tendermint.com/master/) and the [official paper](https://arxiv.org/abs/1807.04938) about the consensus algorithm.   
+
+## Consensus algorithm
+
+Tendermint is a partially synchronous BFT consensus protocol. The protocol requires a fixed set of validators that attempt to come to consensus on one value or, more precisely, on a block (a block is a list of transactions) using gossip-based communication. The consensus algorithm proceeds in rounds: in each round there is a validator that proposes a value (proposer) and all the validators then vote during the round on whether to accept the proposed value or move on to the next round (proposers are chosen according to their voting power).
+
+As most of the consensus algorithms, the Tendermint consensus algorithm assumes a correct majority of processes: less than one third of processes are faulty (*f*), therefore the total number of processes *n* is strictly greater than *3f* (*n > 3f*). For simplicity, we can asssume that:
+ 
+    n = 3f + 1
+
+During a round, processes can exchange different messages and we can summarize the execution in two voting steps: Prevote and Precommit steps. A vote can be for a particular value or for a null value.
+
+A correct process decides on a value *v* in a round *r* upon receiving a proposal and *2f + 1* quorum of Precommit messages for *v* in a round *r*. 
+A correct process sends a Precommit message for a value *v* in a round *r* upon receiving a proposal and *2f + 1* quorum of Prevote messages for *v* in a round *r*.
+
+Validators wait some time before sending a Prevote with a null value if they do not receive a valid proposal after a certain time and they send a Precommit message with a null value if they do not receive *2f + 1* Prevote messages for a value.
+If a correct process receives more than 2/3 Precommit messages for a null value in the same round, it moves to the next round.
```

Comment:
Hmm does this appear in the [algorithm](https://arxiv.org/pdf/1807.04938.pdf)? I do not recall this detail.

Reply:
It doesn't appear explicitly in the algorithm but I thought it was a direct consequence from the fact that, if a process p received 2f + 1 prevotes for nil, a precommit for nil is sent. I changed to make more clear what I wanted to say.